sktalk/corpus/corpus.py

```diff
@@ -84,3 +85,22 @@ def _fromdict(cls, fields):
     @classmethod
     def from_xml(cls, path):
         return XmlFile(path).parse()
+
+    @property
+    def metadata_df(self):
+        """Return the corpus metadata as a pandas dataframe."""
+        if not hasattr(self, "_metadata_df"):
```

Comment:
As suggested in `conversation.py`, I would explicitly set `self._metadata_df = None` in `__init__.py`, and replace this check with:
```suggestion
        if self._metadata_df is None:
```